prov/rxm/src/rxm_attr.c

```diff
@@ -36,7 +36,7 @@ struct fi_tx_attr rxm_tx_attr = {
 	.caps = FI_MSG | FI_TAGGED | FI_SEND,
 	.comp_order = FI_ORDER_STRICT,
 	.inject_size = RXM_TX_DATA_SIZE,
-	.size = 1024,
+	.size = SIZE_MAX,
```

Comment:
Yes, this would get replaced with underlying provider's tx context size in rxm_info_to_rxm().